webview/src/stories/Setup.stories.tsx

```diff
@@ -8,6 +8,10 @@ import { App } from '../setup/components/App'
 const DEFAULT_DATA: SetupData = {
   canGitInitialize: false,
   cliCompatible: true,
+  dvcCliDetails: {
+    exampleCommand: 'path/to/python -m dvc',
```

Comment:
should this be `invocation` instead of `exampleCommand`? If not let's match it to the UI (command). Again, I know I am going directly against https://github.com/iterative/vscode-dvc/issues/3434#issuecomment-1516777571 but I think we could find a better word in the UI for "Command".

Reply:
I'll update it to `command` for now!